oe_paragraphs.install

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the OE Paragraphs module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\Core\Entity\Entity\EntityFormMode;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Copy the values to the new variant field and delete the old variant fields.
+ */
+function oe_paragraphs_update_8001(array &$sandbox) {
+  if (!isset($sandbox['total'])) {
+    // Initialise batch variables and settings.
+    $sandbox['variant_fields'] = [
+      'oe_content_row' => 'field_oe_content_row_variant',
+      'oe_list_item_block' => 'field_oe_list_item_block_variant',
+      'oe_list_item' => 'field_oe_list_item_variant',
+    ];
+    $paragraph_ids = [];
+
+    foreach ($sandbox['variant_fields'] as $bundle => $field) {
+      // Make sure we have the old fields in the database.
+      $all_bundle_fields = \Drupal::service('entity_field.manager')->getFieldDefinitions('paragraph', $bundle);
+      if (!isset($all_bundle_fields[$field])) {
+        continue;
+      }
+
+      // Take all entities that has a value in one of the old fields.
+      $query = \Drupal::entityQuery('paragraph');
+      $query->exists($field);
+      $result = $query->execute();
+      if (!empty($result)) {
+        $paragraph_ids += $result;
+      }
+
+      $sandbox['total'] = count($paragraph_ids);
```

Comment:
I think we can set this values after going through all the items as oppose to setting them 3 times.